Source/WTF/wtf/JSONValues.cpp

```diff
@@ -492,16 +525,55 @@ RefPtr<Value> Value::asValue()
 
 RefPtr<Object> Value::asObject()
 {
+    switch (m_type) {
+    case Type::Null:
+    case Type::Boolean:
+    case Type::Double:
+    case Type::Integer:
+    case Type::String:
+    case Type::Array:
+        return nullptr;
+    case Type::Object:
+        static_assert(sizeof(Object) == sizeof(ObjectBase), "cannot cast");
```

Comment:
No need for this string. Not sure it adds anything.

Reply:
Dropped.